front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/ChartWidgets/messages.ts

```diff
@@ -67,4 +67,30 @@ export default defineMessages({
     id: 'app.containers.Admin.reporting.components.ReportBuilder.Widgets.ChartWidgets.ActiveUsersWidget.ChartWidgetSettings.youNeedToSelectADateRange',
     defaultMessage: 'You need to select a date range first.',
   },
+  analyticsChartAriaLabel: {
+    id: 'app.containers.admin.ReportBuilder.charts.analyticsChartAriaLabel',
+    defaultMessage: 'Label',
+  },
+  analyticsChartDescription: {
+    id: 'app.containers.admin.ReportBuilder.charts.analyticsChartDescription',
+    defaultMessage: 'Description',
+  },
+  accessibility: {
+    id: 'app.containers.admin.ReportBuilder.charts.accessibility',
+    defaultMessage: 'Accessibility',
+  },
+  analyticsChartAriaLabelTooltip: {
+    id: 'app.containers.admin.ReportBuilder.charts.analyticsChartAriaLabelTooltip',
+    defaultMessage:
+      'Provide a label that describes the chart for screen reader users.',
+  },
+  analyticsChartDescriptionTooltip: {
+    id: 'app.containers.admin.ReportBuilder.charts.analyticsChartDescriptionTooltip',
+    defaultMessage:
+      'Provide a more detailed description of the chart for screen reader users.',
```

Comment:
Since this description will also be visible in the UI, could we update this to something like:

_"Provide a more detailed description of the chart and any relevant trends for improved accessibility"._

Also, when you update copy after it's been committed to your branch, you'll also have to change the key on line 88 so it registers as a "new" version:

E.g. Update it to:
id: 'app.containers.admin.ReportBuilder.charts.analyticsChartDescriptionTooltip2',